sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala

```diff
@@ -107,14 +114,14 @@ class Telemetry(serviceName: String, system: ActorSystem) {
         .extract(OtelContext.current(), metadata, otelGetter.asInstanceOf[TextMapGetter[Object]])
       val tracer = openTelemetry.getTracer("java-sdk")
       val span = tracer
-        .spanBuilder(s"""${service.serviceName}.${command.entityId}""")
+        .spanBuilder(s"""${command.entityId}""")
         .setParent(context)
         .setSpanKind(SpanKind.SERVER)
         .startSpan()
       Some(
         span
           .setAttribute("service.name", s"""${service.serviceName}.${command.entityId}""")
-          .setAttribute(s"kalix.${service.componentType}", command.entityId))
+          .setAttribute(s"${service.componentType}", command.entityId))
```

Comment:
Isn't it strange to have an attribute key defined by the user/unknown up front?

Vs something like:
```
.setAttribute("component.type", service.componentType)
.setAttribute("entity.id", command.entityId)
```

If we don't know the attribute name up front, how can it be used anywhere?

Reply:
Good point

Reply:
done